src/generated/type/src/model.rs

```diff
@@ -1184,39 +1184,39 @@ impl PostalAddress {
 #[non_exhaustive]
 pub struct Quaternion {
     /// The x component.
-    pub x: f64,
+    pub x: Option<f64>,
```

Comment:
I am not sure this is the code we want.  Seems like the annotations are creating more trouble than it is worth.

Maybe we should discuss what specific problem the non-optional fields caused with pagination?

Reply:
Yes, I was honoring the logic that states we should treat things as optional if not specified, but I think this was too far reaching as it should only apply to the request messages -- from my understanding. When I reduced the logic to ensure the required annotation was being strictly honored it produced no diffs. May look at this again in the future but I agree it is best to just close this for now.